README.md

```diff
@@ -703,11 +698,22 @@ startup’s growth and how that affects who owns the stock.
   For this reason, the equity team members get is higher in the earlier stages (starting
   with founders) and increasingly lower as a company matures.
   (See the picture above.)
+- 🄳 At some point early on, generally before the first employees are hired, stock will be
```

Comment:
Consider movies defining occurrence of option pool to Stocks and Shares.

Reply:
Prefer to leave it here for now; perhaps it can move but the option pool is something that is created (and can change) in early stages of a company, not a fundamental concept like stock and shares (that section).